pkgs/desktops/gnome/core/evolution-data-server/default.nix

```
@@ -32,6 +32,7 @@
 , ninja
 , libkrb5
 , openldap
+, enableOAuth2 ? lib.meta.availableOn stdenv.hostPlatform webkitgtk_4_1 && !webkitgtk_4_1.meta.broken
```

```suggestion
, enableOAuth2 ? lib.meta.availableOn stdenv.hostPlatform webkitgtk_4_1
```
as discussed on matrix. meta.broken shouldn't really change what features are enabled.

---

This way, `webkitgtk_4_1.meta.platforms` needs to be updated to `platforms.linux`. Is it okay?

---

Since it theoretical works on darwin it should have set broken and we just use false here.

```suggestion
, enableOAuth2 ? stdenv.isLinux
```

---

Done.